tests/unit/models/transactions/test_did_set.py

```diff
@@ -0,0 +1,54 @@
+from unittest import TestCase
+
+from xrpl.models.exceptions import XRPLModelException
+from xrpl.models.transactions import DIDSet
+
+_ACCOUNT = "r9LqNeG6qHxjeUocjvVki2XR35weJ9mZgQ"
+_VALID_FIELD = "1234567890abcdefABCDEF"
+_TOO_LONG_FIELD = "A" * 257
+_BAD_HEX_FIELD = "random_non_hex_content"
+_BAD_HEX_TOO_LONG_FIELD = "r" * 257
```

Comment:
nit: r is an odd character to repeat 257 times for hex codes
```suggestion
_BAD_HEX_TOO_LONG_FIELD = "r" + "a" * 256 // Total 257 chars
```

Reply:
Oh wait, nevermind hahaha - I thought the "Bad" in bad hex was just referring to the length and got confused that the `r` was because of `r`-addresses (which aren't hex to begin with).

Reply:
I'll change it to a different non-hex character.